gooddata-sdk/tests/catalog/test_catalog_permission.py

```diff
@@ -54,6 +56,12 @@ def _assert_default_permissions(catalog_declarative_permissions: CatalogDeclarat
     assert set(permission.name for permission in catalog_declarative_permissions.permissions) == {"ANALYZE", "VIEW"}
 
 
+def _assert_organization_permissions_id(
+    catalog_organization_permissions: [CatalogDeclarativeOrganizationPermission],
```

Comment:
This is not correct typing. Note that `mypy` does not work in tests, so it is undetected. Please use `List[CatalogDeclarativeOrganizationPermission]`.

Reply:
I missed this one, thank you